libraries/classes/Controllers/Database/Operations/CollationController.php

```diff
@@ -43,7 +43,7 @@ public function __invoke(ServerRequest $request): void
             return;
         }
 
-        if (empty($_POST['db_collation'])) {
+        if (empty($request->getParsedBodyParam('db_collation'))) {
```

Comment:
You should assign the value to a variable instead of calling `getParsedBodyParam` every time.
```suggestion
        $dbCollation = $request->getParsedBodyParam('db_collation');
        if (! is_string($dbCollation) || $dbCollation === '') {
```

Reply:
missed that, updated, thx